.github/workflows/template-build.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  workflow_call:
+    inputs:
+      release-tag-name:
+        required: true
+        type: string
+      push:
+        required: true
+        type: boolean
+env:
+  PLATFORMS: linux/amd64,linux/arm64
+
+jobs:
+  dapper-build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Run dapper
+        run: make default
+
+      - name: Read Secrets
+        uses: rancher-eio/read-vault-secrets@main
+        if: ${{ inputs.push == true }}
+        with:
+          secrets: |
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_PASSWORD
```

Comment:
Not sure whether we need to change this like 
https://github.com/harvester/terraform-provider-harvester/blob/4e07ed02cba6fe7fc37bc0d41ff433df81e69915/.github/workflows/build.yml#L58-L59.

We can merge it first and see whether we can get credential. Thanks.

Reply:
Not sure which one is right... 
`secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials` or  `secret/data/github/repo/${{ github.repository }}/dockerhub/harvester/credentials`
I saw some of repos like [load-balancer-harvester](https://github.com/harvester/load-balancer-harvester/blob/0025da293f105cc500a27795249e4f952d80adcd/.github/workflows/template-build.yml#L40-L41), [vm-import-controller](https://github.com/harvester/vm-import-controller/blob/0628775de7840f6780bb5dc2b2238e1c66581b7b/.github/workflows/template-build.yml#L37-L38) use the former one.
Maybe both of them works.

Reply:
c.c. @Yu-Jack

Reply:
I asked this question before here https://github.com/rancherlabs/eio/issues/2153#issuecomment-2074150242. I think `rancher` is enough although I'm not sure why `harvester` works in https://github.com/harvester/terraform-provider-harvester.

Reply:
Got it. Thanks for the information.